README.md

```diff
@@ -19,7 +19,7 @@
 
 ## Overview
 
-`fused-effects` is an effect system for Haskell emphasizing expressivity and efficiency. The former is achieved by encoding [algebraic](#algebraic-effects), [higher-order](#fused-effects) effects, while the latter is the result of [fusing](#fusion) effect handlers all the way through computations.
+`fused-effects` is an effect system for Haskell emphasizing expressivity and efficiency. The former is achieved by encoding [algebraic](#algebraic-effects), [higher-order](#higher-order-effects) effects, while the latter is the result of [fusing](#fusion) effect handlers all the way through computations.
```

Comment:
Great catch! Looks like I was over-eager with my find/replace when I renamed the package 😆

Reply:
I only noticed because I tapped on the link to see if it was an external link, or a link to somewhere later in the readme. 🤷‍♂️